starfish/experiment/builder/orderediterator.py

```diff
@@ -31,12 +31,12 @@ def join_dimension_sizes(
 
 
 def ordered_iterator(
-        dimension_sizes: Sequence[Tuple[Indices, int]]
+        dimension_ids: Sequence[Tuple[Indices, Sequence[int]]]
 ) -> Iterator[Mapping[Indices, int]]:
     """
-    Given a sequence of tuples of dimensions and its respective size, return an iterator that steps
-    through all the possible points in the space.  The sequence is ordered from the slowest varying
-    dimension to the fastest varying dimension.
+    Given a sequence of tuples of dimensions and its respective sequence of ids, return an iterator
```

Comment:
I'm a little confused about the terminology, are ids different then labels? Maybe I should wait till the glossary though : )